frontend/app-development/features/latestCommit/LatestCommit.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { StudioPageSpinner } from '@studio/components';
+import { gitCommitPath } from 'app-shared/api/paths';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { useBranchStatusQuery } from 'app-development/hooks/queries';
+import { useTranslation } from 'react-i18next';
+
+export const LatestCommit = (): React.ReactElement => {
```

Comment:
Maybe we should call this component `LatestCommitInGiteaPage` or something similar to emphasize that it is a page that opens in gitea?